### PR TITLE
Display monsters (ditto)

### PR DESCRIPTION
**Pull request by LegitDongo**
## Description
Add new MAD-based "display" monster information
tl;dr, the monster that gets shown in the overworld when it's a ditto

Note: these keys are subject to change since nothing has really been approved

It adds some nice DTS

`display_mon_id` - the ID of the monster being displayed
`display_mon_name`  - the name of the displayed monster (based on locale)
`display_mon_id_3` - a pre-padded zero 3-digit number representing the displayed monster ID
`display_mon_id_2` - a pre-padded zero 2-digit number representing the displayed monster ID
`display_costume_id` - the ID of the costume the displayed monster has
`display_costume_id_2` - a pre-padded zero 2-digit number representing the displayed monster's costume ID
`display_costume_id_3` - a pre-padded zero 3-digit number representing the displayed monster's costume ID
`display_costume` - the full name of the displayed monster's costume (based on locale)
`display_form_id` - the ID of the displayed monster's form
`display_form` - the name of the displayed monster's form (based on locale)
`display_form_id_3` - a pre-padded zero 3-digit number representing the displayed monster's form ID
`display_form_id_2` - a pre-padded zero 2-digit number representing the displayed monster's form ID
`display_gender` - the greek symbol representing the displayed monster's gender

Since we need a way to use the "display" monster image for the default image provider, I added some alarm keys for monsters too

Discord: `use_display_icon` and `use_display_avatar`
FB: `use_display_image`
Slack: `use_display_icon`
Telegram: `use_display_sticker`

!! this gets overridden if the key exists for a custom image provider !!

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context
Some people want to know which monster to click on

## How Has This Been Tested?
win 10, python 3.8, discord only

## Wiki Update
- [x] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

Wiki changes to come later based on team discussion, see above for full listing of new keys

### CODE REVIEW

**Review by CarlosVMX1 — APPROVED** [review]
Had been using this PR for some weeks and all works good

**Comment by billyjbryant** [comment]
Is this intended to show what the Ditto is displaying as rather than "Ditto"? Can you show a screenshot of what this looks like in use?

File: PokeAlarm/Alarms/Telegram/TelegramAlarm.py [hunk 1]
```diff
@@ -205,7 +208,9 @@ def generic_alert(self, alert, dts):
         message = replace(alert.message, dts)
         lat, lng = dts['lat'], dts['lng']
         max_attempts = alert.max_attempts
-        sticker_url = replace(alert.sticker_url, dts)
+        sticker_url = replace(alert.display_sticker_url, dts) \
+            if alert.use_display_sticker is False\
+            else replace(alert.sticker_url, dts)
```
Comment (billyjbryant) [inline PokeAlarm/Alarms/Telegram/TelegramAlarm.py]: Is this section working as intended? From my reading, it is using `alert.display_sticker_url` if `alert.use_display_sticker` is `False`. Isn't it supposed to be the other way around?

**Comment by LegitDongo** [comment]
> Is this intended to show what the Ditto is displaying as rather than "Ditto"? Can you show a screenshot of what this looks like in use?

Indeed; Can't show a screenshot unfortunately, but here's the associated PR
https://github.com/Map-A-Droid/MAD/pull/1198

File: PokeAlarm/Alarms/Telegram/TelegramAlarm.py (hunk 1 above)
Comment (LegitDongo) [reply PokeAlarm/Alarms/Telegram/TelegramAlarm.py]: You are correct, I must have missed this

**Review by billyjbryant — APPROVED** [review]

**Review by billyjbryant — CHANGES_REQUESTED** [review]
Can you fix that invalid truthy evaluation?

**Review by billyjbryant — APPROVED** [review]
